src/configuration.ts

```diff
@@ -170,7 +172,8 @@ export function getElementAdjustment(elementName: string): ColorAdjustment {
 
 export function getElementStyle(
   backgroundHex: string,
-  elementName?: string
+  elementName?: string,
+  includeBadgeStyles: boolean = false
```

Comment:
`: boolean` was not needed since `false` made it a boolean.

Reply:
Sure. I've noticed you tend to favor the type inferences and I'll try to stick to that. I am relatively new to TypeScript and I think I am just so excited to have types that I get overly type happy.

Reply:
No worries.